packages/core/src/services/webhooks/listWebhooks.ts

```diff
@@ -0,0 +1,12 @@
+import { eq } from 'drizzle-orm'
+
+import { database } from '../../client'
+import { webhooks } from '../../schema/models/webhooks'
+import { type Webhook } from './types'
+
+export async function listWebhooks(
+  workspaceId: number,
+  db = database,
+): Promise<Webhook[]> {
+  return db.select().from(webhooks).where(eq(webhooks.workspaceId, workspaceId))
+}
```

Comment:
i'm actually choosing not to use repositories because all the troubles we've experienced so far. My counter proposal are basic services to do gets.